src/jarviscg/cli.py

```diff
@@ -14,9 +14,9 @@
 
 import argparse
 import time
-import formats
-from jarvis import CallGraphGenerator
-from utils.constants import CALL_GRAPH_OP, KEY_ERR_OP
+from . import formats
+from .core import CallGraphGenerator
+from .utils.constants import CALL_GRAPH_OP, KEY_ERR_OP
```

Comment:
You are mixing and matching relative imports and package imports (used for example in src/jarviscg/formats/fasten.py).

Not a problem, just a style thing.

(note: you can do `from ..format` to import a module from a level above).

Reply:
Thanks for mentioning this! Looks like [PEP 8](https://peps.python.org/pep-0008/#imports) says:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured

So I went with absolute imports in the files I touched in https://github.com/nuanced-dev/jarviscg/pull/12/commits/75a835b5b062a75a191a4d98b529c9fcac62198b